nautobot/extras/signals.py

```diff
@@ -281,7 +281,7 @@ def refresh_job_models(sender, *, apps, **kwargs):
 
         for module_details in modules.values():
             for job_class in module_details["jobs"].values():
-                # TODO: catch DB error in case where multiple Jobs have the same grouping + name
+                # TODO(Glenn): catch DB error in case where multiple Jobs have the same grouping + name
```

Comment:
What clarification are you looking for here?

Reply:
Was just adding your name to it. :)